common/am/rest-api/src/main/java/org/apache/syncope/common/rest/api/service/wa/SAML2SPKeystoreService.java

```diff
@@ -7,7 +7,7 @@
  * "License"); you may not use this file except in compliance
  * with the License.  You may obtain a copy of the License at
  *
- *   http://www.apache.org/licenses/LICENSE-2.0
+ *    http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Same as above

Reply:
@mmoayyed was this overlooked by your last commit?

Reply:
I think so. I am going to pass through all components and make sure formatting is right.

Reply:
Pushed another commit to fix formatting issues.